app/assets/javascripts/services/notification_svc.js

```diff
@@ -1,6 +1,7 @@
 samson.service('notificationService', function(messageCenterService) {
-  this.info = function(message, options) {
+  var _this = this;
```

Comment:
usually this is labelled 'self' instead of '_this'. I think I picked that up from the 'Best parts of JS' book.